docker/mainnet/dash/dash.mainnet.conf

```diff
@@ -0,0 +1,11 @@
+port=10333
+rpcport=10332
+
+rpcallowip=172.100.2.100
```

Comment:
Should we put comment who is allowed to make rpc call here?

Reply:
Yes, nice idea.